src/storm/storage/prism/Program.cpp

```diff
@@ -707,6 +707,19 @@ storm::expressions::Expression Program::getInitialStatesExpression() const {
     }
 }
 
+bool Program::hasIntervalUpdates() const {
```

Comment:
Broader discussion: We currently do not cache such information, which may be a downside for some of the larger prism programs. Seems not urgent for me, but wanted to know your thoughts.

Reply:
We'd have to make sure that the cached information remains valid (or cleared) whenever the program is changed. 

Imo, there needs to be evidence of this being a practical performance issue before making the code more complex / unsafe by introducing a caching mechanism.

Reply:
I think programs are basically thought to be immutable for us, right? At least,most transformations create new objects.  

I think part of the discussion here is that the QVBS does not have large symbolic programs; but that is a choice of the benchmarks. 

Anyhow, not urgent.